package.json

```diff
@@ -819,7 +819,7 @@
     "ts-loader": "^7.0.5",
     "ts-morph": "^9.1.0",
     "tsd": "^0.13.1",
-    "typescript": "4.1.3",
+    "typescript": "4.3.5",
```

Comment:
@tylersmalley we need to update prettier as well. see https://prettier.io/blog/2021/05/09/2.3.0.html#support-typescript-42-10418httpsgithubcomprettierprettierpull10418-10466httpsgithubcomprettierprettierpull10466-10546httpsgithubcomprettierprettierpull10546-10589httpsgithubcomprettierprettierpull10589-by-sosukesuzukihttpsgithubcomsosukesuzuki

maybe it's better to update to the latest version in a separate PR, as it will affect lots of code in the repo

Reply:
also, you might need to update babel to support syntax introduced in the latest TS versions